jetty-home/src/main/resources/modules/demo.d/demo-login.conf

```diff
@@ -0,0 +1,17 @@
+ee10-xyz {
+org.eclipse.jetty.ee10.jaas.spi.PropertyFileLoginModule required
+debug="true"
+file="${jetty.base}/etc/ee10-demo-login.properties";
+};
+
+ee9-xyz {
+org.eclipse.jetty.ee9.jaas.spi.PropertyFileLoginModule required
+debug="true"
+file="${jetty.base}/etc/ee9-demo-login.properties";
+};
+
+ee8-xyz {
+org.eclipse.jetty.ee8.jaas.spi.PropertyFileLoginModule required
+debug="true"
+file="${jetty.base}/etc/ee8-demo-login.properties";
+};
```

Comment:
Why is a core demo having eeX configuration?

Reply:
If you look at  `jetty-home/src/main/resources/modules/demo.d/` you will see lots of other eeX related config files.